modules/virt-example-snippets-nmstate-IP-management.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * virt/node_network/virt-updating-node-network-config.adoc
+
+[id="virt-example-snippets-nmstate-IP-management_{context}"]
+= Example snippets: IP management
+
+The following are example configuration snippets for different methods of IP management. 
+
+These snippets use the `ethernet` interface type to simplify the example while showing the related context in the Policy configuration. These IP management examples can be used with the other interface types. 
+
+== Static
+
+The following snippet creates an ethernet interface that uses a static IP address:
+
+[source,yaml]
+----
+...
+interfaces:
+- name: eth0
+  type: ethernet
+  state: up
+  ipv4:
+    address:
+    - ip: 192.168.122.250 <1>
+      prefix-length: 24
+    enabled: true
+...
+----
+<1> Replace with the static IP address for the interface.
+
+== No IP address
+
+The following snippet creates an ethernet interface that has no IP address:
+
+[source,yaml]
+----
+...
+interfaces:
+- name: eth0
+  type: ethernet
+  state: up
+  ipv4:
+    enabled: false
+...
+----
+
+== Dynamic host configuration
+
+The following snippet creates an ethernet interface that uses a dynamic IP address, gateway address, and DNS:
+
+[source,yaml]
+----
+...
+interfaces:
+- name: eth0
+  type: ethernet
+  state: up
+  ipv4:
+    dhcp: true
+    enabled: true
+...
+----
+
+The following snippet creates an ethernet interface that uses a dynamic IP address but without a dynamic gateway address or DNS:
+
+[source,yaml]
+----
+...
+interfaces:
+- name: eth0
+  type: ethernet
+  state: up
+  ipv4:
+    dhcp: true
+    auto-gateway: false
+    auto-dns: false
+    enabled: true
+...
+----
+
+== DNS
+
+The following snippet creates an ethernet interface that 
+
+[source,yaml]
+----
+...
+interfaces:
+   ...
+dns-resolver:
+  config:
+    search:
+    - example.com
+    - example.org
+    server:
+    - 8.8.8.8
+...
+----
+
+== Static routing
+
+The following snippet creates an ethernet interface that 
```

Comment:
The following snipped configures a static route and a static IP on interfaces eth1. Note that the next-hop-address must be reachable through one of the interfaces.